ops/charm.py

```diff
@@ -780,11 +783,11 @@ def restore(self, snapshot: Dict[str, Any]):
 
 
 class SecretExpiredEvent(SecretEvent):
-    """Event raised by Juju on the owner when a secret's expiration time elapses.
+    """Event triggered by Juju on the owner when a secret's expiration time elapses.
 
     This event is fired on the secret owner to inform it that the secret revision
     must be removed. The event will keep firing until the owner removes the
```

Comment:
maybe add a note of how long it will keep firing (forever?) and at what kind of interval

Reply:
I'm not sure we should go into too much detail here. That should be clearly documented by Juju (though it probably isn't!) but the Juju docs seem to me the place to put details like that.